pol-core/pol/module/uomod.cpp

```diff
@@ -4783,7 +4782,14 @@ bool UOExecutorModule::is_reserved_to_me( Item* item )
 
 BObjectImp* UOExecutorModule::mf_Shutdown()
 {
-  Clib::exit_signalled = true;
+  int exit_code;
+
+  if ( !exec.hasParams(1) || !getParam( 0, exit_code ) )
+  {
+    exit_code = 0;
+  }
```

Comment:
I feel like I have to think a lot about whether `exit_code` will always be initialized.  So a suggestion:

```suggestion
  int exit_code = 0;

  if ( exec.hasParams(1) )
  {
    getParam( 0, exit_code );
  }
```

Also, we are effectively ignoring the error code of `getParam`, and this calls this out.